pkg/pillar/types/contenttreetypes.go

```diff
@@ -146,23 +150,19 @@ func (status ContentTreeStatus) IsContainer() bool {
 	return false
 }
 
-// IsOCIRegistry will return true if datastore is an OCI registry
-func (status ContentTreeStatus) IsOCIRegistry() bool {
-	if status.DatastoreType == zconfig.DsType_DsContainerRegistry.String() {
-		return true
-	}
-	return false
-}
-
```

Comment:
The intention was to go over the list once (or at least on each datastore update if we take this async path), and not each time we want to check the OCI registry. But probably here not a big deal

Reply:
The other reason to keep this field as cached is another morbid fields: `AllDatastoresResolved` and `DatastoresTypesList', so that we traverse list once (or at least minimize traversing) and calculate both: `IsOCIRegistry` and `AllDatastoresResolved`. 

But I feel that new fields are somehow not welcomed in these types of structures. What is the rule of thumb?